### DISCUSSION TITLE
Custom rsuite table Column with default props (i.e. flexGrow={1})

### DISCUSSION THREAD

**Discussion by UTkzhang** [rsuite/rsuite-table#344] [Q&A]
hi @simonguo, wondering how I can override the default values on table Columns. I tried to create a component and export e.g. <Column {...defaultProps}/>, but it seems like this is ignored by rsuite Table. Is Column not customizable? Thanks!

**Comment by simonguo** [comment]
Fixed in https://github.com/rsuite/rsuite-table/pull/388
